Resources/Storage/StorageBase.py

```diff
@@ -229,7 +229,7 @@ def getCurrentURL( self, fileName ):
     if not result['OK']:
       return result
     cwdUrl = result['Value']
-    fullUrl = '%s/%s' % ( cwdUrl, fileName )
+    fullUrl = '%s%s' % ( cwdUrl, fileName )
```

Comment:
I think this fix is not correct. I guess this is done to avoid double slash in the fullUrl since in the only place it is used the lfn ( starting with '/' ) is passed.  In the FTSClient, where this is used, we should rather use constructURLFromLFN( lfn ) method of a storage element. getCurrentURL() was meant rather to be used to "navigate" the SE namespace.